sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/DiagnosticsClientContext.java

```diff
@@ -87,6 +88,11 @@ public void serialize(DiagnosticsClientContext clientContext, JsonGenerator gene
         private RntbdTransportClient.Options options;
         private String rntbdConfigAsString;
         private ConnectionMode connectionMode;
+        private String machineId;
+
+        public void withMachineId(String machineId) {
+            this.machineId = machineId;
```

Comment:
I notice in tests, we are adding prefix `"vmId:"`, if that's the prefix we always want to add, worth adding it here? 
like we have done in `setMachineId()` ?

Reply:
"vmId" would be used if the identifier comes from VM Instance metadata - uuid: prefix if it is the static identifier